src/components/BookComponent/AddBook.js

```diff
@@ -1,21 +1,52 @@
-import React from 'react';
+import React, { useState } from 'react';
+import { useDispatch } from 'react-redux';
+import { v4 as uuidv4 } from 'uuid';
+// import BookPage from '../../BookPage';
```

Comment:
- [ ] **[OPTIONAL]** You can remove the commented code to make your codebase cleaner.